contracts/core/SetToken.sol

```diff
@@ -286,8 +284,6 @@ contract SetToken is
         public
         returns (bool)
     {
-        // Confirm address is not null
```

Comment:
Any reason we remove this

Reply:
It's duped...that logic already exists in the StandardToken contract which we call through `super`.